utils/documentOps.py

```diff
@@ -41,17 +41,17 @@
 #   where <op> is the ONNX op name
 #   where <text> is optional text, currently unused
 #
-# ==LIM== <text> 
+# ==LIM== <text>
 #   where <text> qualifies the current restrictions to the implementation.
 #
 # ==TODO== <text>
-#   where <text> add "private" info about what needs to be fixed. 
+#   where <text> add "private" info about what needs to be fixed.
 #
 # ==MIN== <num>
 #   where <num> is the minimum release version supported.
 #
-# ==MAX== <num>
-#   where <num> is the maximum release version supported.
+# ==UNSUPPORTED== <num>
```

Comment:
An assumption of the code is that we must see MIN before UNSUPPORTED. It could be removed (by putting the init code in the OP which by def needs to come first).

I am ok as is, just indicate that MIN must be precede UNSUPPORTED.

Reply:
Murphy's law says that if I even try a comment update, the build system will break because it knows I'm leaving for vacation in a few hours.

I created https://github.com/onnx/onnx-mlir/issues/2460 for myself to cover this when I get back ;-)